snippets/spring-data/transactions.json

```diff
@@ -0,0 +1,32 @@
+{
+    "Transaction Example": {
+        "prefix": ["@cbsdtranex"],
+        "body": ["package ${1:package name};","","import org.springframework.beans.factory.annotation.Autowired;","import org.springframework.data.couchbase.core.CouchbaseTemplate;","import com.couchbase.client.java.transactions.TransactionResult;","","public class transactionExample {","","    @Autowired private CouchbaseTemplate CouchbaseTemplate;","    ","    public void main(String[] args) {","   ","        TransactionResult result = CouchbaseTemplate.getCouchbaseClientFactory().getCluster().transactions().run(ctx -> {","            Model found = CouchbaseTemplate.findById(${2:Model}.class).inScope(\"${3:scopeName}\").inCollection(\"${4:collectionName}\").one(${5:documentId});","        });","","    }","    ","}"],
```

Comment:
The class here starts with a lower case "t".

And it has same issues as a previous example - I don't think it will work as there's no Spring app scaffolding or Couchbase configuration.  Looks like this example wasn't actually run.

It would be nice if the transactions example was multi-document.